app/src/main/java/com/android/feedme/model/viewmodel/CameraViewModel.kt

```diff
@@ -43,10 +46,18 @@ class CameraViewModel : ViewModel() {
   // Could be useful later if lastPhoto needs to be accessed outside the view model
   val lastPhoto = _lastPhoto.asStateFlow()
 
+  val listOfIngredient: List<Ingredient> =
+      listOf(Ingredient("Tomate", "DEFAULT_TYPE", "DEFAULT_ID"))
```

Comment:
Are you sure to keep this default ingredient tomato here ? I thin it was probably meant for testing ?

Reply:
Yes, it was just for testing :)